hydra/test_utils/launcher_common_tests.py

```diff
@@ -346,6 +349,15 @@ def sweep_two_config_groups(
 
 @pytest.mark.usefixtures("hydra_restore_singletons")
 class IntegrationTestSuite:
+
+    def post_run_callback(
+        self, **kwargs: Any
+    ) -> Optional[Callable[[Dict[Any, Any]], None]]:
+        """
+        This will be called after running test app and before validating output file from test.
+        """
+        return None
+
```

Comment:
I would like this function to have a clear signature. this is an API we expect other plugins to use.

Reply:
added signature.